stratum/lib/security/testdata/certs1/ca.crt

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
+MIICbTCCAdYCCQDznEzuzqQGcjANBgkqhkiG9w0BAQsFADB7MQswCQYDVQQGEwJV
+UzELMAkGA1UECAwCQ0ExEzARBgNVBAcMCk1lbmxvIFBhcmsxIzAhBgNVBAoMGk9w
+ZW4gTmV0d29ya2luZyBGb3VuZGF0aW9uMRAwDgYDVQQLDAdTdHJhdHVtMRMwEQYD
+VQQDDApTdHJhdHVtIENBMB4XDTIxMTEwNDE4NDQ1OFoXDTMxMTEwMjE4NDQ1OFow
+ezELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAkNBMRMwEQYDVQQHDApNZW5sbyBQYXJr
+MSMwIQYDVQQKDBpPcGVuIE5ldHdvcmtpbmcgRm91bmRhdGlvbjEQMA4GA1UECwwH
+U3RyYXR1bTETMBEGA1UEAwwKU3RyYXR1bSBDQTCBnzANBgkqhkiG9w0BAQEFAAOB
+jQAwgYkCgYEA3rD8R7NBxL3HdNaSBnNJYfnn93k8bZmmCuYCop5M48o+nUhNhkF2
+qwroM/FnEa1uKstlLTRX4I2cngownBuMzk43rGX3gwPd+T6AvoY9I9yjzCTvYR+z
+3bGGTBgGFyDRAsyqplqoA4dazrDt/aV2o1O2Ux4Y3jS5yNYMUBElv88CAwEAATAN
+BgkqhkiG9w0BAQsFAAOBgQBoXWWJMmpjMvx+aCK3PTCstfSCYt2Vptw9NUZ4c1CA
+uGLiYwZ8tbMbaA9K8G6lQLiapM+Hx60KiTyG8tMzSANDZWKLGYt6A1+vFqaTcP/H
+KCmvUsDKNsKffOHg57olHzCE/Rtcp2xzepybZ9Kkl2WMrzGdZG/IPdxp9rQURlxB
+KQ==
+-----END CERTIFICATE-----
```

Comment:
These should be generated on the fly. Either using the openssl API or with a bazel rule calling the `openssl` binary.

Reply:
These are generated on the fly now. Let me know what you think